### PR TITLE
chore(deps): update gcds-components, gcds-tokens, gcds-utility packages

### PR DESCRIPTION
**Pull request by daine**
# Summary | Résumé
Updates the packages to the latest versions

[English preview](https://pr-443.djtlis5vpn8jd.amplifyapp.com/)
[French preview](https://pr-443.d35vdwuoev573o.amplifyapp.com/)

Note: Update token package again before merging.

### CODE REVIEW

File: src/_includes/partials/token_table.njk [hunk 1]
```diff
@@ -37,14 +37,15 @@
...
-                        <span lang="en">--gcds-{{ concatToken | replace: ".", "-" | fixTokenName }}-{{ item[0] }}</span>
+                        <span lang="en">--gcds-{{ concatToken | replace: ".", "-" | fixTokenName | camelToKebab }}-{{ item[0] | camelToKebab }}</span>
                     {%- when 'value' %}
                         {{- item[1].value }}
                     {%- when 'contrast' %}
                         {{- item[1].value | contrast }}
                     {%- when 'rgb' %}
                         {{- item[1].value | hexTo: 'rgb' }}
                     {%- when 'px' %}
+                        PX
```
Comment (daine) [inline src/_includes/partials/token_table.njk:48]:
```suggestion
```
I think this isn't supposed to be here?
Screen Shot 2024-12-10 at 10 22 26 PM

Comment (melaniebmn) [reply src/_includes/partials/token_table.njk]: Oops good catch! That was from testing and I forgot to remove it - thank you!

**Review by ethanWallace — DISMISSED** [review]
LGTM!

**Review by daine — COMMENTED** [review]
I looked at the changes, LGTM! I can't approve though since I'm the original PR author

**Review by ethanWallace — APPROVED** [review]
LGTM!

**Comment by daine** [comment]
Typing my approval here ✅

**Pull request merged**